match/lib/match/commands_generator.rb

```diff
@@ -155,7 +155,8 @@ def run
         FastlaneCore::CommanderGenerator.new.generate(Match::Options.available_options, command: c)
 
         c.action do |args, options|
-          Match::Migrate.new.migrate(args, options)
+          params = FastlaneCore::Configuration.create(Match::Options.available_options, options.__hash__)
+          Match::Migrate.new.migrate(params)
```

Comment:
Why this change was needed? Was it to allow for direct calls of this command via Ruby?

Reply:
To be able to call it from the tests:

https://github.com/fastlane/fastlane/pull/20776/files/868db12e6781cffc6347a31fee4e2ad84eb31167#diff-a3fda74b644561adcfae676cad5b8ca95f7ee0d7edea8ebcc54c359926e118c2R51

Otherwise, this:

https://github.com/fastlane/fastlane/pull/20776/files/868db12e6781cffc6347a31fee4e2ad84eb31167#diff-f79e3f0b905679ef7e333b8a455ee64c7b44ca4ecab8ca7cf487d479ef5f33dbL11

fails with an error stating that there is no `__hash__` method or something.